Solutions/Authomize/Hunting queries/Admin_SaaS_account_detected.yml

```diff
@@ -0,0 +1,37 @@
+id: b3430fb5-78aa-4729-8595-f66c06138478
+name: Admin SaaS account detected
+description: |
+  'detects internal admins accounts, it's recommended to review any new administrative permission.'
+requiredDataConnectors:
+  - connectorId: Authomize
+    dataTypes:
+      - Authomize_v2_CL
+tactics:
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1089
+query: |
+  Authomize_v2_CL
```

Comment:
same comments as mentioned in analytics rule

Reply:
done